include/ddc/kernels/splines/matrix.hpp

```diff
@@ -69,6 +108,14 @@ class Matrix
         return b;
     }
 
+    /**
+     * @brief Solve the multiple-right-hand-side linear problem Ax=b inplace.
+     *
+     * @param[in, out] bx A 2D mdpsan storing the multiple right-hand-sides of the problem and receiving the corresponding solution. Important note: the convention is the reverse of the common matrix one,
+     * row number is second index and column number the first one. It should be changed in the future.
```

Comment:
"It should be changed in the future."

Not sure if this should be said with such certainty. We are solving the problem:
Ax_i=b_i
where i is the column index. In most algorithms, ideally we want to be contiguous within each sub-equation so this is the correct indexing. Maybe it is possible to clear this up with Kokkos Views and left v right layout but as written this sounds like DDC thinks it is important to move to a less optimal solution.

Reply:
`ideally we want to be contiguous within each sub-equation so this is the correct indexing`, as long as we use Kokkos::View/mdspan we have no link between ordering of indexes and contiguousity/storage. So to me it should not let you think this will make us move to a less optimal solution. I think this is a Fortran biais ^^

Reply:
Ah ok, sorry I didn't realise that mdspan also had a left/right layout parameter. In that case I agree with you